src/server.rs

```diff
@@ -71,9 +75,11 @@ impl ServerBuilder {
 
 /// OPC UA server.
 ///
-/// This represents an OPC UA server. Nodes can be added through the several methods below, and then
-/// the server can be started with [`run()`](Self::run).
-pub struct Server(ua::Server);
+/// This represents an OPC UA server. Nodes can be added through the several methods below.
+///
+/// Note: The server must be started with [`ServerRunner::run()`] before it is actually visible to
+/// external clients.
+pub struct Server(Arc<ua::Server>);
```

Comment:
Shouldn't this `Server` handle be `Clone`able, other than its counterpart `ServerRunner` which is supposed to be consumed by running it? The API ensures that all access to the inner, shared `ua::Server` is thread-safe. The contents of the server might be controlled by different, concurrent worker threads.